.github/workflows/release.yaml

```diff
@@ -0,0 +1,52 @@
+name: 'Release'
+ # Version bump and release
+ on:
+   workflow_run:
+     workflows: ["Validate"]
+     type:
+       - complete
+
+ jobs:
+   bump-version:
```

Comment:
I don't think `bump-version` represents what this job does. What do you thing?

Reply:
Renamed to bump-and-tag. Either way, that's a key, the name that gets displayed is "Package, Bump Version and Tag"